objects/ooptionicons/Step_0.gml

```diff
@@ -31,7 +31,7 @@ if (P1menuRight && menuCooldown < 1 && image_index = 0)
 		musicVolume += 0.01;
 	}
 	
-		audio_group_set_gain(audiogroup_music, musicVolume, 0);
+		audio_group_set_gain(audiogroup_mus, musicVolume, 0);
```

Comment:
Make sure the indentation is lined up with everything else in the if statement. It makes the code look more neat.